packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -65,6 +65,10 @@ TODO: Remove this section if there are not any general updates.
 
 TODO: Remove this section if there are not any general updates.
 
+## VS Code Sidebar updates
+
+* When using VS Code with a light theme, the embedded sidebar provided by DevTools will now also show in the light theme
+
```

Comment:
please link this PR here in the release notes. thanks!

Reply:
Oops, done!